src/figuresClasses.ts

```diff
@@ -1,19 +1,63 @@
+/* eslint-disable max-len */
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
```

Comment:
We should reuse our interface Figure to type other classes.

Please use `class Triangle implements Figure`

Also for other classes